libs/community/langchain_community/document_loaders/mongodb.py

```diff
@@ -66,7 +68,22 @@ async def aload(self) -> List[Document]:
                 "database": self.db_name,
                 "collection": self.collection_name,
             }
-            result.append(Document(page_content=str(doc), metadata=metadata))
+            if self.field_names is not None:
```

Comment:
Is there a way to provide the field names as part of the request to mongo instead? So we don't end up sending those fields over the wire if we don't need them.